### PR TITLE
Add ruby-lint support

### PR DESCRIPTION
**Pull request by adamors**
Add rubylint maker

### CODE REVIEW

File: autoload/neomake/makers/ft/ruby.vim [hunk 1]
```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#ruby#EnabledMakers()
-    return ['mri', 'rubocop', 'reek']
+    return ['mri', 'rubocop', 'reek', 'rubylint']
```
Comment (wsdjeg) [inline autoload/neomake/makers/ft/ruby.vim]: I think we should not return this makers without executable() check;
see #375 
@blueyed 
I think `EnabledMakers()` should only return the effective maker in current os;
for example
```vim
function! neomake#makers#ft#ruby#EnabledMakers()
      let makers = []
      if executable('ruby-lint')
           call add(makers,'ruby-lint')
      endif
      return makers    
endf
```

Comment (adamors) [inline autoload/neomake/makers/ft/ruby.vim]: Except MRI, every Ruby maker needs an existing executable.

Comment (wsdjeg) [inline autoload/neomake/makers/ft/ruby.vim]: and also you can provide a g:var for user to set the executable.

Comment (blueyed) [inline autoload/neomake/makers/ft/ruby.vim]: @wsdjeg 
The "executable" check is done in https://github.com/neomake/neomake/blob/1b585580f2f1d33dc115ebf86466bbb4112ed8da/autoload/neomake/utils.vim#L97-L107 (via `neomake#utils#MakerIsAvailable`) already.  But this should probably use Vim's `executable` however.
Creating a PR for that.  In this PR here it's fine.

File: README.md [hunk 2]
```diff
@@ -113,6 +113,7 @@ Ruby:
 - mri
 - jruby
 - rubocop
+- rubylint
 - reek
```
Comment (blueyed) [inline README.md]: I think the order should reflect the actual one from `neomake#makers#ft#ruby#EnabledMakers`.

File: autoload/neomake/makers/ft/ruby.vim (hunk 1 above)
Comment (blueyed) [inline autoload/neomake/makers/ft/ruby.vim]: @wsdjeg 
> and also you can provide a g:var for user to set the executable.

That would be `g:neomake_ruby_rubylint_exe` (already supported).  See https://github.com/neomake/neomake/blob/1b585580f2f1d33dc115ebf86466bbb4112ed8da/doc/neomake.txt#L186-L190.

**Comment by blueyed** [comment]
LGTM.

@adamors 
Please squash it into a single commit, although we can do so when merging (but you have more control over the commit message then).

**Comment by adamors** [comment]
Okay.

**Comment by blueyed** [comment]
Thanks, @adamors!

**Pull request merged**